flow_action_components/DedupeRecordCollection/force-app/main/default/classes/DedupeRecordCollection.cls

```diff
@@ -0,0 +1,49 @@
+global with sharing class DedupeRecordCollection {
```

Comment:
Looking at this class here, it feels like it now belongs in the CollectionProcessors project. how do feel about moving it there?

Reply:
I would still want to package this project with the two recalc actions.

Reply:
@adwhite75 I'm sensitive to checkin in this dedupe because we've got those two other dedupes under discussion at https://github.com/alexed1/LightningFlowComponents/pull/475, and I'm worried about name collisions and the like. Can you rationalize this situation? How do we get to a single dedupe that lives in the collection processors package and has the union of functionality of these three variants?

Reply:
I'd agree it belongs in CollectionsProcessors - I would also agree that there isnt a need for this, https://github.com/alexed1/LightningFlowComponents/pull/475, and ExtractStrings' dedupe functionality. I'm fine with this one or Mike Gill's going. I actually didnt mean to merge this one into master, I was just uploading the code and stupidly made it a PR :)

Reply:
> Looking at this class here, it feels like it now belongs in the CollectionProcessors project. how do feel about moving it there?

Let me get on that! I lost track of this one.

Reply:
I am going to remove the DeDupe action from this branch and just provide the code to Mike, then move this into CollectionProcessors